doc/source/parameters/cfd/boundary_conditions_multiphysics.rst

```diff
@@ -0,0 +1,55 @@
+==================================
+Multiphysics - Boundary Contitions
+==================================
+
+This subsection's purpose is defining the boundary conditions associated to multiphysic problems. 
+
+Heat Transfer
+^^^^^^^^^^^^^
+
+.. code-block:: text
+
+    subsection boundary conditions heat transfer
+    set number                  = 2
+        subsection bc 0
+            set type	        = temperature
+            set value	        = 0
+        end
+        subsection bc 1
```

Comment:
I think it is a good idea to devide two boundary conditions for two possibilities! Only thing, to standardize with the other documentation, we could write the default parameters. They are 0 both Tinf and h.